src/PrestaShopBundle/Entity/ModuleHistory.php

```diff
@@ -0,0 +1,174 @@
+<?php
+
+namespace PrestaShopBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * ModuleHistory
+ *
+ * @ORM\Table
+ * @ORM\Entity
+ * @ORM\HasLifecycleCallbacks
+ */
+class ModuleHistory
+{
+    /**
+     * @var integer
+     *
+     * @ORM\Column(name="id", type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var integer
+     *
+     * @ORM\Column(name="id_employee", type="integer")
+     */
+    private $idEmployee;
+
+    /**
+     * @var integer
+     *
+     * @ORM\Column(name="id_module", type="integer")
+     */
+    private $idModule;
+
+    /**
```

Comment:
This is for consistency with legacy since PrestaShop has always used ```date_add``` and ```date_upd``` fields in its DB